bootstrap/sql/postgresql/v001__create_tables.sql

```diff
@@ -22,7 +22,6 @@ CREATE TABLE IF NOT EXISTS schema_metadata_info (
   "schemaGroup"     VARCHAR(255) NOT NULL,
```

Comment:
in v0001 should all be VARCHAR(256) 
change to VARCHAR(255) should be in a v002/3